src/btree/bt_split.c

```diff
@@ -1597,12 +1594,6 @@ __wt_multi_to_ref(WT_SESSION_IMPL *session, WT_PAGE *page, WT_MULTI *multi, WT_R
         break;
     }
 
-    /*
-     * There can be an address or a disk image or both, but if there is neither, there must be a
-     * backing lookaside page.
-     */
-    WT_ASSERT(session, multi->has_las || multi->addr.addr != NULL || multi->disk_image != NULL);
```

Comment:
I think this assert is still valid in durable history. In my branch,  I just changed it to 
WT_ASSERT(session, multi->addr.addr != NULL || multi->disk_image != NULL);

Reply:
6bc31ef